examples/cluster/plot_color_quantization.py

```diff
@@ -61,7 +61,7 @@
 print("done in %0.3fs." % (time() - t0))
```

Comment:
Since we are modifying the example, do you mind improving the other `print` with the f-string.

Reply:
Done ;)